.github/workflows/ci.yml

```
@@ -0,0 +1,137 @@
+name: MO4 Coding Standard CI
+on:
+  pull_request:
+  push:
+  schedule:
+  - cron: '30 5 * * 1'
```

Feels a bit too often for me, how about this?
```suggestion
  - cron: '30 5 1 * *'
```